packages/nimble-components/src/rich-text/editor/template.ts

```diff
@@ -84,12 +87,31 @@ export const template = html<RichTextEditor>`
                         ${x => richTextToggleNumberedListLabel.getValueFor(x)}
                         <${iconNumberListTag} slot="start"></${iconNumberListTag}>
                     </${toggleButtonTag}>
+                    ${repeat(
+        x => Array.from(x.getMentionExtensionConfig()),
```

Comment:
Isn't mentionExtensionConfig already an array?
```suggestion
        x => x.getMentionExtensionConfig(),
```

Reply:
Updated the logic as per the suggestion